spec/rubocop/cop/rails/validation_spec.rb

```diff
@@ -8,22 +8,35 @@
 
   described_class::BLACKLIST.each_with_index do |validation, number|
     it "registers an offense for #{validation}" do
-      inspect_source(cop,
-                     "#{validation} :name")
+      inspect_source(cop, "#{validation} :name")
       expect(cop.offenses.size).to eq(1)
     end
 
     it "outputs the correct message for #{validation}" do
-      inspect_source(cop,
-                     "#{validation} :name")
+      inspect_source(cop, "#{validation} :name")
       expect(cop.offenses.first.message)
         .to include(described_class::WHITELIST[number])
     end
   end
 
   it 'accepts new style validations' do
-    inspect_source(cop,
-                   'validates :name')
+    inspect_source(cop, 'validates :name')
     expect(cop.offenses).to be_empty
   end
+
+  it 'autocorrect validates_presence_of' do
```

Comment:
Of course, I can add a similar validation with a different name, but don`t think they will be needed (from the point of the test). In fact the name is not important.
Below we check to change the old validation with parameters.